lib/player.rb

```diff
@@ -0,0 +1,14 @@
+class Player
+  attr_accessor :name, :mark_spots, :current_player
```

Comment:
- [ ] The only attr_accessor for this class should be :current_player the others could be attr_readers and your game still works correctly